README.md

```diff
@@ -43,7 +43,25 @@
 
 Check our additional materials and our documentation guide [here](https://sap.github.io/cloud-sdk/docs/dzen/getting-started )
 
-## Guidelines on documentation style
+## How we write
+
+- Being concise is a virtue. Write short and on purpose.
+- Provide only relevant context. Make a good assumption of readers' knowledge and make it your starting point for context.
+- Give a link whenever you mention 3rd party resources, library, concept, abbreviation, reference, etc.
+- Avoid abbreviations or introduce them unless it's absolutely obvious and widely adopted.
+- Build upon already available content and use it for reference. Create materials that could be leveraged by others.
+  - Think modular. Extract logically independent parts and link to them from larger scope materials.
+  - Use the same approach you do with coding when structuring your application
+  - Increase scalability so that others can leverage your work
+  - Generalize up to the level where it makes sense but not more
+- Ask if your peers understood what you meant
+- Be your reader, make it a candy
+- Don't sell, don't do marketing, the value comes from the content itself, not you words of accolade to our work. Leave that for Blogs and even there be humble.
```

Comment:
typo: you->use.

Reply:
I think I wanted to write "your words" :))) I anyway want to improve this phrase to sound better. It's verbose and badly worded now. It reminds me of another suggestion that when we see an obvious fix let's use suggestion button. I nicely highlight the proposal and allow for an immediate commit in case of full agreement.